src/users/usersActions.js

```diff
@@ -25,17 +26,41 @@ export const reportPresence = (isActive: boolean = true, newUserInput: boolean =
   });
 };
 
-const typingWorker = auth => ({
-  get_current_time: () => new Date().getTime(),
+const typingWorker = (state: GlobalState) => {
+  const auth: Auth = getAuth(state);
 
-  notify_server_start: (user_ids_array: number[]) => {
-    api.typing(auth, JSON.stringify(user_ids_array), 'start');
-  },
+  const getRecipients = user_ids_array => {
+    // User ID arrays are only supported in server versions >= 2.0.0-rc1, for
+    // versions before this, email arrays are used. If current server version is
+    // undetermined, user ID arrays are optimistically used.
+    let useEmailArrays: boolean;
+    const activeAccount: Account | void = tryGetActiveAccount(state);
 
-  notify_server_stop: (user_ids_array: number[]) => {
-    api.typing(auth, JSON.stringify(user_ids_array), 'stop');
-  },
-});
+    if (!activeAccount || activeAccount.zulipVersion === undefined) {
```

Comment:
`activeAccount?.zulipVersion === undefined`, I think.

Reply:
If I use `?.`, flow complains below because `activeAccount` is `Account | void`, i.e I can't use `activeAccount.zulipVersion` then.

Reply:
Well, that's an interesting bug! Not much to be done about it for now.

For future reference, it appears to specifically be `github.com/facebook/flow/issues/6613`, which was fixed in v0.114.0. (At time of writing, the latest version of React Native is v0.62.2, using Flow v0.113.0.)